pkg/build/api/validation/validation.go

```diff
@@ -64,14 +64,15 @@ func ValidateBuildConfig(config *buildapi.BuildConfig) field.ErrorList {
 	fromRefs := map[string]struct{}{}
 	specPath := field.NewPath("spec")
 	triggersPath := specPath.Child("triggers")
+	buildFrom := buildutil.GetImageStreamForStrategy(config.Spec.Strategy)
```

Comment:
depends...when would that etcd content go back through validation?  certain "oc edit bc" could be broken (they'd have to fix the BC before it would save).

Is there any other time?

if so then yes, any BC that has a default ICT but does not have the right "From" type would fail validation when previously it did not (but previously the ICT had no effect).